src/util/fees.cpp

```diff
@@ -29,16 +33,33 @@ std::string StringForFeeReason(FeeReason reason) {
     return reason_string->second;
 }
 
-bool FeeModeFromString(const std::string& mode_string, FeeEstimateMode& fee_estimate_mode) {
-    static const std::map<std::string, FeeEstimateMode> fee_modes = {
-        {"UNSET", FeeEstimateMode::UNSET},
-        {"ECONOMICAL", FeeEstimateMode::ECONOMICAL},
-        {"CONSERVATIVE", FeeEstimateMode::CONSERVATIVE},
-    };
-    auto mode = fee_modes.find(mode_string);
+static const std::vector<std::pair<std::string, FeeEstimateMode>> FEE_MODES = {
+    {"unset", FeeEstimateMode::UNSET},
+    {"economical", FeeEstimateMode::ECONOMICAL},
+    {"conservative", FeeEstimateMode::CONSERVATIVE},
+    {(CURRENCY_UNIT + "/kB"), FeeEstimateMode::BTC_KB},
+    {(CURRENCY_ATOM + "/B"), FeeEstimateMode::SAT_B},
```

Comment:
We can't use `CURRENCY_*` here in global scope, since the initialisation order is  undefined. A simple static function wrapper should work.

Reply:
Odd that no compiler warning or anything was emitted about it. I've updated the code to wrap the map.

Reply:
For future reference, the original: 3bad9a438769c6749063ab5ab405a62f6efea8cc